infinispan-spring-boot-starter-embedded/src/main/java/org/infinispan/spring/starter/embedded/InfinispanConfigurationCustomizer.java

```diff
@@ -2,6 +2,7 @@
 
 import org.infinispan.configuration.cache.ConfigurationBuilder;
 
+@Deprecated
```

Comment:
javadoc?
is it going to be replaced? dropped?

Reply:
droped